src/backend/base/langflow/load/load.py

```diff
@@ -13,7 +14,7 @@
 from langflow.utils.util import update_settings
 
 
-def load_flow_from_json(
+async def load_flow_from_json(
```

Comment:
DO we need to keep a sync version ?

Reply:
I think we do for the sake of BC but all the code that uses this function can use the async one.

Reply:
OK. Done, PTAL.